react/features/base/dialog/components/web/StatelessDialog.js

```diff
@@ -116,8 +122,11 @@ class StatelessDialog extends Component<Props> {
         return (
             <Modal
                 autoFocus = { true }
+                components = {{
+                    Header: customHeader ?? undefined
```

Comment:
do you really need to do this? if the prop is not given it will be undefined...

Reply:
IIRC when I was testing I needed to explicitly pass that but I could be mistaken, will double check and alter if that's not the case